### DISCUSSION TITLE
Slint for Ecommerce Site

### DISCUSSION THREAD

**Discussion by MohaBeacon** [slint-ui/slint#6548] [Q&A]
Am just wondering is it a feasible idea to use slint to build a ecommerce site

**Comment by hunger** [comment]
I am pretty sure it could be done, but in a web context Slint will be one canvas element that is used to render the entire slint UI, implemented in one WASM binary. I am not sure I would want that as a web-first application, as that makes a lot of "web things" hard to do- E.g. accessibility will be as Slint does it, not as other web apps do it, linking to sub pages becomes harder, a lot of web technologies will not be available and you will need to add crates to your WASM to support some functionality that a browser would handle for web apps. You will be better of with some web framework I think.

That could of course look different when you need a Slint UI for some (low powered?) device and just want some web-viewable page to show that same UI for remote access.
